pkg/scheduler/algorithm/priorities/image_locality_test.go

```diff
@@ -31,7 +31,7 @@ func TestImageLocalityPriority(t *testing.T) {
 	test40250 := v1.PodSpec{
 		Containers: []v1.Container{
 			{
-				Image: "gcr.io/40",
+				Image: "gcr.io/40:v1",
```

Comment:
You should still have "gcr.io/40" test case and validate this priority can find imageSize instead of un-match

Reply:
I see - the rationale was "gcr.io/140",  "gcr.io/250", and "gcr.io/2000" in the three test suites respectively are still untagged - so a mixture of tagged and untagged images should have good coverage.

Reply:
E.g.,
```
test40250 := v1.PodSpec{
		Containers: []v1.Container{
			{
				Image: "gcr.io/40:v1",
			},
			{
				Image: "gcr.io/250",
			},
		},
	}
```